frontend/views/partials/createbookclub.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
```

Comment:
add section identation @KarimElSheikh 
like 
please add section identifier to know what begins where
like
<!-edit form starts here-->
Html code
<!-edit form ends here-->